apps/system-e2e/src/tests/judicial-system/smoke/custody-prosecutor.spec.ts

```diff
@@ -64,13 +64,15 @@ test.describe('Custody Prosecutor', () => {
     await page.locator('input[id=reqValidToDate]').fill(today)
     await page.keyboard.press('Escape')
     await page.locator('input[id=reqValidToDate-time]').fill('16:00')
+    await page.waitForTimeout(1000)
```

Comment:
Instead of waiting for some arbitrary time, why not wait for the thing you're waiting for? Are you waiting for a HTTP response? A UI element update?

Reply:
Honestly a lack of familiarity with Playwright on my part coupled with impatience to test out our hypothesis 😊 I'll have a look in to seeing if I can implement this in a different way.